kiosk/components/PaymentForm.tsx

```diff
@@ -21,9 +21,12 @@ const PaymentForm = (props: PaymentFormProps): JSX.Element => {
       return;
     }
 
+    // Encode email to preserve any special characters (e.g. +, &)
+    const encodedEmail = encodeURIComponent(email);
```

Comment:
Unfortunately, this new code does not work either.

input: jel+dummy@sitecore.net
Confirmation page URL: https://jel-pull0516-2-kiosks.sitecoredemo.com/payment/confirmed/1?email=jel%2Bdummy%40sitecore.net
QR code URL: https://jel-pull0516-2-website.sitecoredemo.com/?email=jel+dummy@sitecore.net